src/log.cpp

```diff
@@ -53,8 +75,14 @@ void Log::debug(const char *fmt, ... )
     vsnprintf( buf, sizeof(buf) - 1, fmt, vl );
     va_end( vl );
 
-    // Add the log line
-    g()->add( buf );
+    Log * self = g();
+
+    if (self->mOutputFile == nullptr) {
+        // Add the log line
+        self->add( buf );
+    } else {
```

Comment:
We have to add to buffer even if logging into file is enabled, so the user can see the log output using a button.